app/api/db/__init__.py

```diff
@@ -12,12 +12,15 @@
 import api.logging
 from api.db.db_config import DbConfig, get_db_config
 from api.db.migrations.run import have_all_migrations_run
+from api.db.models import init_lookup_tables
 
 logger = api.logging.get_logger(__name__)
 
 
 def init(
-    config: Optional[DbConfig] = None, check_migrations_current: bool = False
+    config: Optional[DbConfig] = None,
+    sync_lookups: bool = False,
```

Comment:
nit: rename to `sync_lookup_tables` but let's first resolve the discussion about whether to use lookup tables in the first place